src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -7471,6 +7473,26 @@ class BodyVisitor : public CommonVisitor<BodyVisitor> {
                 ASR::ttype_t *type = ASRUtils::TYPE(ASR::make_Pointer_t(al, x.base.base.loc, type_));
                 tmp = ASR::make_GetPointer_t(al, x.base.base.loc, args[0].m_value, type, nullptr);
                 return ;
+            } else if( call_name.substr(0, 6) == "bitnot" ) {
+                parse_args(x, args);
+                if (args.size() < 1 or args.size() > 1) {
```

Comment:
You cannot use `or` in C++, it fails with MSVC:
```suggestion
                if (args.size() < 1 || args.size() > 1) {
```

Reply:
Also why not just:
```suggestion
                if (args.size() != 1) {
```

Reply:
Got it. Thank you for the guidance. I will update it.